README.md

```diff
@@ -1,46 +1,98 @@
-The PHP Interpreter
-===================
+<div align="center">
+    <a href="https://php.net">
+        <img
+            alt="PHP"
+            src="https://static.php.net/www.php.net/images/logos/new-php-logo.svg"
+            width="150">
+    </a>
+</div>
 
-This is the github mirror of the official PHP repository located at
-https://git.php.net.
+# The PHP Interpreter
 
-[![Build Status](https://secure.travis-ci.org/php/php-src.svg?branch=master)](http://travis-ci.org/php/php-src)
+PHP is a popular general-purpose scripting language that is especially suited to
+web development. Fast, flexible and pragmatic, PHP powers everything from your
+blog to the most popular websites in the world. PHP is distributed under the PHP
+License v3.01.
+
+[![Build status](https://travis-ci.org/php/php-src.svg?branch=master)](https://travis-ci.org/php/php-src)
 [![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy?svg=true)](https://ci.appveyor.com/project/php/php-src)
 
-Pull Requests
-=============
-PHP accepts pull requests via github. Discussions are done on github, but
-depending on the topic can also be relayed to the official PHP developer
-mailing list internals@lists.php.net.
+## Documentation
+
+The PHP manual is available online at https://php.net/docs.
+
+## Installation
```

Comment:
I'd reword the first section as follows:

Before:

```markdown
## Installation

There are many ways to use PHP. For the installation of PHP, please refer to the
online documentation available at https://php.net/install.

### Prebuilt packages and binaries

Prebuilt packages and binaries can be used to get up and running fast with PHP.

For Windows, the PHP binaries can be obtained from https://windows.php.net.
After extracting the archive the `*.exe` files are ready to use.

For other systems see the documentation.
```

After:

```markdown
## Installation

### Prebuilt packages and binaries

Prebuilt packages and binaries can be used to get up and running fast with PHP.

For Windows, the PHP binaries can be obtained from https://windows.php.net.
After extracting the archive the `*.exe` files are ready to use.

For other systems see the documentation at https://php.net/install.
```

Reply:
Sounds good, @javiereguiluz. Applied. Thank you.